languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -19621,6 +19621,329 @@ USA
+                <pattern>
+                    <token postag="SENT_START|CC|PCT" postag_regexp="yes">
+                        <exception regexp="yes">as|for|if</exception>
+                    </token>
+                    <token min="0" skip="-1" chunk="B-NP-singular" >
+                        <exception regexp="yes">i|those</exception>
+                    </token>
+                    <token chunk="E-NP-singular">
+                        <exception postag="CD" />
+                        <exception regexp="yes">few|.*s|some|stuff|they|.*thing|those|></exception>
+                    </token>
+                    <token regexp="yes">that|which|who</token>
+                    <token min="0" postag="RB" />
+                    <token min="0" postag="MD" />
+                    <token skip="-1" chunk_re=".-VP">
+                        <exception scope="current" postag="MD|RB" postag_regexp="yes" />
+                        <exception scope="current" regexp="yes">be|can|i|.*nt|to</exception>
+                        <exception scope="next" postag="DT|MD|NN(S|PS?)?|PCT|PRP.*|VBZ" postag_regexp="yes" />
+                        <exception scope="next" regexp="yes">and|cant|gonna|to|wont</exception>
+                    </token>
+                    <marker>
+                        <token postag="VBP" chunk_re=".-VP">
+                            <exception regexp="yes">cause|found|spoke</exception>
+                            <exception regexp="yes" inflected="yes">be|end|please|will|&uncommon_verbs;</exception>
+                            <exception regexp="yes" case_sensitive="yes">[A-Z].*</exception>
+                        </token>
+                    </marker>
+                    <token><exception>to</exception></token>
+                </pattern>
+                <message>The verb form '\8' does not seem to match the subject.</message>
+                <suggestion><match no="8" postag="VBZ" /></suggestion>
+                <suggestion><match no="8" postag="VBD" /></suggestion>
+                <example correction="lists|listed">The file which was sent today <marker>list</marker> all of the topics for the meeting.</example>
+                <example correction="keeps|kept">The guy who didn't study <marker>keep</marker> staring at my notes.</example>
+                <example correction="annoys|annoyed">The woman who works here really <marker>annoy</marker> me.</example>
+                <example correction="declares|declared">That zoo that operated illegally finally <marker>declare</marker> bankruptcy.</example>
+                <example correction="loses|lost">The rookie athlete who doesn't practice <marker>lose</marker> most of his games.</example>
+                <example correction="shows|showed">The scan that is running currently <marker>show</marker> that some malware has been detected.</example>
+                <example correction="leaved|leaves|left">The guy that knows <marker>leave</marker> this morning.</example>
```

Comment:
To fix the test, all you need to do is fix the order of suggestions: `leaves|leaved|left` 🤓

Reply:
About the "leaved" thing – at some point, we might have AI/ML help in determining whether the context allows for the plant-related sense of "to leave". "The trees have leaved to soon this spring". Might be an idea to mention during the workshop next week.

Reply:
![Screen Shot 2021-06-16 at 2 27 29 PM](https://user-images.githubusercontent.com/81176099/122296391-347ae500-ceaf-11eb-96c9-b289ca96c46e.png)

El
Oh
El

Reply:
Ex dee, indeed. Y "leaf", tho? The lemma is different ("leaves" should not be a VBZ of the supposed verb "leaf")!

Reply:
XD